### ISSUE TITLE
v18.09.0-beta3 wget: bad address 'repo.tinycorelinux.net'

### ISSUE THREAD

**Issue by DieterRogiest** [boot2docker/boot2docker#1338]
On OS X I downloaded boot2docker v18.09.0-beta3 and created a vm with it:
```
docker-machine create --virtualbox-boot2docker-url=/Users/dieter/.docker/machine/cache/boot2docker.iso --driver=virtualbox vbox-test2

docker-machine ls
vbox-test2   -        virtualbox   Running   tcp://192.168.99.100:2376           v18.09.0-beta3
```

In Virtualbox then this vm appears where I can access its console or start it.

Following https://github.com/butomo1989/docker-android/blob/master/README_VMWARE.md
I try in the vm:

```
tce-load -wi kvm
wget: bad address 'repo.tinycorelinux.net'

curl http://www.google.com
curl: (6) Could not resolve host: www.google.com
```

No google solution to this problem. restarting docker service doesn't help.
ONLY ONCE after sudo reboot did it succeed to connect to google.com and repo.tinycorelinux.net! 2 minutes later in the same session it failed again.

My VM created with the previous release version https://github.com/boot2docker/boot2docker/releases/tag/v18.06.1-ce never has this problem.

(I am not using VMWare Fusion because I had no clue how to use it to start the vm that is in /Users/dieter/.docker/machine/machines.)

**Comment by tianon** [comment]
This sounds like a problem with your environment -- can you provide any more details like the contents of `/etc/resolv.conf`, the output of `ip address`, whether things like `nslookup google.com 1.1.1.1` can resolve `google.com` appropriately, etc?

Did you try the previous version in VirtualBox?  Mixing VirtualBox and VMware on the same system isn't something I've ever had good results with, so that could potentially be related to your issues.

**Comment by tianon** [comment]
I wonder if this is related to what was fixed by https://github.com/boot2docker/boot2docker/pull/1344?

**Comment by tianon** [comment]
(Closing as presumed either fixed by #1344 or an environmental issue.)

**Comment by tianon** [comment]
See https://github.com/boot2docker/boot2docker/releases/tag/v18.09.1-beta1 and https://github.com/boot2docker/boot2docker/releases/tag/v18.09.1-beta2 (which include this fix). :+1:
